ocaml/xapi/context.ml

```diff
@@ -237,15 +237,28 @@ let parent_of_origin (origin : origin) span_name =
 let start_tracing_helper parent_fn task_name =
   let open Tracing in
   let span_details_from_task_name task_name =
-    match String.split_on_char ':' task_name with
-    | [x; uuid] when x = "Export of VM" ->
-        let span_name = x in
-        (span_name, [("xs.span.arg.vm.uuid", String.trim uuid)])
-    | [x; y; uuid] when x = "dispatch" && y = "system.isAlive" ->
-        let span_name = x ^ ":" ^ y in
-        (span_name, [("xs.xapi.rpc.msg.uuid", uuid)])
-    | _ ->
-        (task_name, [])
+    let uuid_length = 36 in
+    let export_of_vm = "Export of VM" in
+    let dispatch_system_is_alive = "dispatch:system.isAlive" in
+    let open String in
+    if starts_with ~prefix:export_of_vm task_name then
+      ( export_of_vm
+      , [
+          ( "xs.span.arg.vm.uuid"
+          , sub task_name (length export_of_vm + 2) uuid_length
+          )
+        ]
+      )
```

Comment:
```suggestion
      let uuid = 
        sub task_name (length export_of_vm + 2) uuid_length
      in
      (export_of_vm, [("xs.span.arg.vm.uuid", uuid)])
```
Is the `+2` there because there's a space between the `:` and the `uuid`?